packages/web-runtime/src/App.vue

```diff
@@ -17,24 +17,51 @@
       <template v-else-if="!showHeader">
         <router-view name="fullscreen" />
       </template>
-      <div v-else key="core-content" class="uk-flex uk-flex-stretch">
+      <div v-else id="web-content" key="core-content" class="uk-flex uk-flex-stretch">
         <transition :name="appNavigationAnimation">
           <focus-trap v-if="isSidebarVisible" :active="isSidebarFixed && appNavigationVisible">
-            <oc-sidebar
+            <oc-sidebar-nav
+              v-show="isSidebarVisible"
+              id="web-nav-sidebar"
               v-touch:swipe.left="handleNavSwipe"
               class="oc-app-navigation"
-              :logo-img="logoImage"
-              :logo-alt="sidebarLogoAlt"
-              :nav-items="sidebarNavItems"
+              :accessible-label="$gettext('Application navigation')"
               :class="sidebarClasses"
-              :fixed="isSidebarFixed"
-              :accessible-label="accessibleLabel"
-              @close="toggleAppNavigationVisibility"
             >
-              <template v-if="sidebar.sidebarFooterContentComponent" v-slot:footer>
+              <template #header>
+                <div class="uk-text-center">
+                  <oc-button
+                    v-if="isSidebarFixed"
+                    variation="inverse"
+                    appearance="raw"
+                    class="web-sidebar-btn-close"
+                    :aria-label="$gettext('Close sidebar navigation')"
+                    @click="toggleAppNavigationVisibility"
+                  >
+                    <oc-icon name="close" aria-hidden="true" />
```

Comment:
The `aria-hidden` is not needed here, since an `oc-icon` without an `accessible-label` has internal logic for setting `aria-hidden="true"`. Can be removed here.